public/js/app/components/statistics/Chart.jsx

```diff
@@ -10,21 +10,34 @@ const colors = {
   grey: '#65656C',
   pink: '#D02F80',
   red: '#B52C17',
+  yellow: '#fab919',
+}
+const schoolsColors = languageIndex => {
+  let { allSchools } = i18n.messages[languageIndex].statisticsLabels
+  const schoolColorsObject = {
+    ABE: colors.red,
+    CBH: colors.grey,
+    EECS: colors.green,
+    ITM: colors.blue,
+    SCI: colors.pink,
+  }
+  allSchools = allSchools.split(' ').join('\n')
+  schoolColorsObject[allSchools] = colors.yellow
+  return schoolColorsObject
 }
-const schoolsColors = { ABE: colors.red, CBH: colors.grey, EECS: colors.green, ITM: colors.blue, SCI: colors.pink }
 
 function countPercentage(numberOfCourses, numberOfDocs) {
   return (Math.abs(numberOfDocs) * 100) / Math.abs(numberOfCourses)
 }
 
-function getChartData(numberName, schools) {
+function getChartData(numberName, schools, languageIndex) {
   const schoolCodes = Object.keys(schools)
 
   return schoolCodes.map(school => {
     const schoolNumbers = schools[school]
     const { numberOfCourses } = schoolNumbers
     return {
-      color: schoolsColors[school],
+      color: schoolsColors(languageIndex)[school],
       school,
```

Comment:
All focus of code should be shifted to school, not to color, which causes a lot of space for future bugs because of code and functions which simply handles just one label which can be avoided if focus moves to school instead of color.